docs/getting-started/installation.rst

```diff
@@ -46,20 +46,16 @@ Installing the "bleeding edge" version from GitHub
 --------------------------------------------------
 
 You can install the very latest, "bleeding edge", development version of
-PyMedPhys by directly cloning its GitHub repo and installing its dependencies
-using the following commands from within a terminal:
+PyMedPhys, but you will first need Git and Yarn. You can find instructions for
+installing Git `here <https://www.atlassian.com/git/tutorials/install-git>`__
+and Yarn `here <https://yarnpkg.com/en/docs/install>`__.
+
+Once you have Git and Yarn, run the following commands in a terminal to clone
+the ``pymedphys`` GitHub repo and install the ``pymedphys`` package along with
+its dependencies:
 
 .. code:: bash
 
     git clone https://github.com/pymedphys/pymedphys.git
     cd pymedphys
-
-    conda config --add channels conda-forge
-    conda install pymedphys --only-deps
-    pip install -e .
-
-
-Note that this requires Git to be installed on your workstation. Instructions
-for installing Git can be found in the `Developer Guide`_.
-
-.. _`Developer Guide`: ../developer/contributing.html#chocolatey
+    yarn bootstrap
```

Comment:
And maybe a "see the developer guide for more detailed instructions on how to set up PyMedPhys for development".

Reply:
Sure :) done!